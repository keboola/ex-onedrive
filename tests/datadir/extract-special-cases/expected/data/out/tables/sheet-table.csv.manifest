{"columns":["Duplicate","Duplicate-1","column-3","column-4","Special_123_uescr","column-6","column-7"]}
