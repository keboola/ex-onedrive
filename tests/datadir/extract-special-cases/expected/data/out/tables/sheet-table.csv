"A","10","(2x empty header)",""," $1,618.50 ","1/1/2014","(empty col)"
"B","20","a",""," $1,321.00 ","1/1/2014",""
"A","30","b",""," $2,178.00 ","1/6/2014","x"
"B","40","a",""," $888.00 ","1/6/2014","y"
"A","50","b",""," $2,470.00 ","1/6/2014","z"
"B","60","a",""," $1,513.00 ","1/12/2014",""
"A","70","b",""," $921.00 ","1/3/2014",""
"","","",""," $2,518.00 ","1/6/2014",""
"","","","","","",""
"","","",""," $1,545.00 ","",""
